apps/openmw/mwworld/cellstore.hpp

```diff
@@ -214,7 +214,7 @@ namespace MWWorld
             /// containers.
             /// @note Triggers CellStore hasState flag.
 
-            Ptr searchConst (const std::string& id) const;
+            ConstPtr searchConst (const std::string& id) const;
             ///< Will return an empty Ptr if cell is not loaded. Does not check references in
             /// containers.
             /// @note Does not trigger CellStore hasState flag. Do not modify the returned Ptr!
```

Comment:
Do we still need that note? It's const so it should be obvious (and compiler enforced) that you should not change the reference.